README.md

```diff
@@ -72,6 +77,7 @@ Here are the main differences between a "pure" maplibre-gl-js map and a Leaflet
 
 - No rotation / bearing / pitch support
 - Slower performances: When using maplibre-gl-leaflet, maplibre-gl-js is set as not interactive. Leaflet receives the touch/mouse events and updates the maplibre-gl-js map behind the scenes. Because maplibre-gl-js doesn't redraw as fast as Leaflet, the map can seem slower.
+- MapLibre restricts the maximum latitude of the map in a stricter way then Leaflet. In order to maximize compatibility it it is recommended to set a `maxBounds: [[180, -Infinity], [-180, Infinity]]` and `maxBoundsViscosity: 1` on your Leaflet `Map` to prevent users from panning past the minimum and maximum latitude supported by MapLibre.
```

Comment:
It looks like our recommendation in the code example and this line disagree?

Do we want to set it to 540 or Infinity? 😄

Reply:
synced this to reflect `Infinity`/`-Infinity` since those are the most permissive bounds that still work and allow for world wrapping.